tests/test_room_routes.py

```diff
@@ -6,7 +6,8 @@
 import sogs.config
 import werkzeug.exceptions as wexc
 from util import pad64
-from request import sogs_get, sogs_post, sogs_put
+from request import sogs_get, sogs_post, sogs_put, sogs_post_raw
+import secrets
```

Comment:
```suggestion
from nacl.utils import random
```
Not a big deal, just a minor preference for libsodium's `randombuf_bytes`, partly because `token_bytes` is a stupid name.